magprime/__init__.py

```diff
@@ -47,6 +47,11 @@ def set_empty_donation(self):
         if not self.extra_donation:
             self.extra_donation = 0
 
```

Comment:
It'd be good to be able to set something like `discount=True` (allow_negatives may be better but discount is more memorable, hm).

Thanks for the examples, makes sense. :)